resources/after.sh

```diff
@@ -67,22 +64,12 @@ displayServerInfo () {
     displayMessage "--------------------------------------"
 }
 
-#
-## Manage current packages
-#
-displayTitle "Update current packages"
-
-dpkg --configure -a --force-confnew
-apt-get -o Dpkg::Options::="--force-confnew" update --fix-missing -y
-
-displayOkMessage
-
 #
 ## Install missing dependencies
 #
 displayTitle "Installing missing dependencies"
 
-apt-get -o Dpkg::Options::="--force-confnew" install -y php7.1-fpm php7.1-bz2 php7.1-mcrypt php7.1-gmp php7.1-xdebug
+apt-get -o Dpkg::Options::="--force-confnew" install -y php7.0-bz2 php7.0-mcrypt php7.0-gmp php7.0-xdebug php7.1-bz2 php7.1-mcrypt php7.1-gmp php7.1-xdebug
```

Comment:
Actually I think mcrypt is deprecated in 7.2 now and in any case we shouldn't be using it so let's get rid?